lib/tapioca/cli.rb

```diff
@@ -311,5 +321,79 @@ def self.exit_on_failure?
         true
       end
     end
+    private
+
+    def print_init_next_steps
+      say("This project is now set up for use with Sorbet and Tapioca.", :bold)
+
+      say(<<~OUTPUT)
+
+      The sorbet/ folder should exist and look something like this:
+
+      ├── config             # Default options to be passed to sorbet on every run
+      └── rbi/
+        ├── annotations/     # Type definitions pulled from the rbi-central repository
+        ├── gems/            # Autogenerated type definitions for your gems
+        └── todo.rbi         # Constants which were still missing after RBI generation
+      └── tapioca/
+        ├── config.yml       # Default options to be passed to tapioca
+        └── require.rb       # A file where you can require any gems that are needed for type generation
+
+      Please check this folder into version control.
+
+      OUTPUT
+
+      say("🤔 What's next?", :bold)
+      say(<<~OUTPUT)
+
+      1. Many Ruby applications use metaprogramming DSLs to dynamically generate constants and methods.
+         To generate type definitions for any DSLs in your application, run:
+
+      OUTPUT
+
+      say("  bin/tapioca dsl", :cyan)
+
+      say(<<~OUTPUT)
+
+      2. Next, check whether things that show up in the sorbet/rbi/todo.rbi file actually exist in your project.
+         It's possible some of these constants are typos.
+
+      3. Finally, you should typecheck your project:
+
+      OUTPUT
+
+      say("  srb tc", :cyan)
+
+      say("\n")
+      say("Documentation", :bold)
+      say("We recommend skimming these docs to get a feel for how to use Sorbet:")
+      say("- Gradual Type Checking: ", nil, false)
+      say("https://sorbet.org/docs/gradual", :cyan)
+      say("- Enabling Static Checks: ", nil, false)
+      say("https://sorbet.org/docs/static", :cyan)
+      say("- RBI Files: ", nil, false)
+      say("https://sorbet.org/docs/rbi", :cyan)
+      say("\n")
+
+      say("Next Steps", :bold)
```

Comment:
The items of this section could be merged after the numbered list at the beginning